constitution.tex

```diff
@@ -794,6 +794,9 @@
 	\item The candidates will be notified of their nomination.
 		Each candidate is given a minimum of twenty-four hour period to accept or decline the nomination.
 		A list of all nominees who have accepted their nominations will be posted shortly thereafter.
+	\item The date and time of the candidate(s)'s speeches will be announced by a member of the Executive Board at least five (5) days prior to the event. 
+		All Active Members are expected to attend this Election, as specified in \ref{Expectations of House Members}.
```

Comment:
I'd lean away from `Election`, it also isn't well defined. You're expecting members to attend speeches or `time before the House to present their platform of candidacy`, correct? I'd specify that, because Election is used differently elsewhere.